### PR TITLE
Updated with details on new locales and quarantine PR

### PR DESCRIPTION
**Pull request by Delphine**
I admit I had a hard time wording this.
Also, I wasn't able to find any specific examples in my archives showing this happening in a PR. I'm assuming because once l10n has been imported, you can't see the l10n.toml file with the locales removed anymore (because they then appear as added in the PR)

### CODE REVIEW

**Review by flodolo — CHANGES_REQUESTED** [review]
New lines are ineffective in markdown, i.e. the text is not displayed as you probably wanted
https://github.com/mozilla-l10n/documentation/blob/347b6884ea9b649969bffadb30104cb6f5ceff8d/src/products/android-l10n/adding_new_locales.md

This is not "probably", the next PR coming from the code repository will try to remove the new locales from l10n.toml until the PR is merged on the other side.

This is one of the cases where it's worth digging deeper to get information, and explain it clearly (with steps) for whoever needs to takeover.
* You merge the PR adding locales to TOML in the l10n repository. What happens next? What does the automation?
* There's a PR on the code side. What happens with the automation while this PR is pending?
* What do you need to do while you wait for the PR to be merged?

**Comment by Delphine** [comment]
Thanks a lot for the feedback. This is not urgent at this point, so I will take the time to investigate and create a doc to address all the questions you mention.

**Comment by Delphine** [comment]
This should be all updated by now!

**Review by flodolo — APPROVED** [review]

**Pull request merged**
